### PR TITLE
Ports Cannabis from TG

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Ports https://github.com/tgstation/tgstation/pull/59360 enough that it works for our code.

- Adds real cannabis, which instead of making you super drugged up, just makes you absolutely baked.
- adds it to ourselves

## Why It's Good For The Game

bluntensmoken

## Changelog

:cl:
add: Cannabis has been introduced to the frontier.
del: The CIA is no longer lacing all of your cannabis product with space drugs
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 1]
```diff
@@ -448,3 +448,50 @@
...
+			M.Sleeping(10 SECONDS)
+		else
+			var/eepy_message = pick("Oh stars you're sleepy", "Your eyelids feel heavy.", "You could use a power nap.", "It'd be nice to lay down a bit....")
+			to_chat(M, span_notice("[eepy_message]"))
+	if(prob(20) && M.buckled && M.body_position != LYING_DOWN && !M.IsParalyzed())
+		to_chat(M, span_warning("It's too comfy to move..."))
+		M.Paralyze(10 SECONDS)
+	return ..()
+
+/datum/reagent/drug/cannabis/puppygirl //the coder's poorly disguised fetish
```
Comment (thgvr) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: no

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 2]
```diff
@@ -448,3 +448,50 @@
...
+	metabolization_rate = 0.125 * REAGENTS_METABOLISM
+
+/datum/reagent/drug/cannabis/on_mob_life(mob/living/carbon/M)
+	M.apply_status_effect(/datum/status_effect/stoned)
+	M.adjust_nutrition(-1 * REM) //munchies
+	if(prob(35))
+		var/smoke_message = pick("You feel relaxed.","You feel calmed.","Your mouth feels dry.","You could use some water.","Your heart beats quickly.","You feel clumsy.","You crave junk food.","You notice you've been moving more slowly.")
+		to_chat(M, span_notice("[smoke_message]"))
+	if(prob(20))
+		M.emote(pick("smile","laugh","giggle"))
```
Comment (thgvr) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: force emotes - no

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm (hunk 1 above)
Comment (Erikafox) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm]: fuck

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm (hunk 2 above)
Comment (Erikafox) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm]: removed

**Pull request closed without merging**